niyonkurueric/My_brand
# #yq7WVfdI CRUD operations

### What does this PR do?
this PR works on CRUD operations of the blog and like for each blog .
### Description of Task to be completed?

- validate the create blog form

- save the created blog in localStorageas an object

- read the blogs from localStorage on the landing page and in the dashboard

- validate update blog form

- read a single blogs from local storage

- save updated blogs in the local storage

- delete an blogs from local storage
- geolocation of queries
### How should this be manually tested?

- git clone and paste the repository link to have the repository to your local machine
- run index inside the folder and check it 
### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
[yq7WVfdI](https://trello.com/b/yq7WVfdI/mybrand)
### Screenshots (if appropriate)
![Screenshot from 2022-01-17 17-57-32](https://user-images.githubusercontent.com/76616593/150519087-60e168c9-56ca-40e8-8cb7-5652b3ebc319.png)
